ncm-network/src/main/pan/components/network/schema.pan

```diff
@@ -10,6 +10,12 @@ include 'quattor/schema';
 
 type component_network_type = {
     include structure_component
+    # Name of the auto-generated udev rule which needs to be removed to allow udev re-consider the interface names
+    "udev_rules_file" ? string
+    # E.g. "/usr/bin/udevadm trigger", although restricting the scope to just network devices should not hurt
+    "udev_trigger_command" ? string
```

Comment:
what possible values could there be?

Reply:
Nowadays I think only what's written in the comments, but I'd need to check. RH4 had an old udev with a different interface, so the value had to be configurable. But the new udevadm CLI is stable for a while, so I'd not mind removing these commands from the templates and just hard-code them in the component.